src/app/[locale]/viewer/_components/submodel-elements/document-component/DocumentComponent.tsx

```diff
@@ -206,7 +212,11 @@ export function DocumentComponent(props: MarkingsComponentProps) {
                 submodelElementPath +
                 '/attachment';
 
-            const imageResponse = await getAttachmentFromSubmodelElement(props.submodelId, submodelElementPath);
+            const imageResponse = await getAttachmentFromSubmodelElement(
+                props.submodelId,
+                submodelElementPath,
+                aasOriginUrl,
```

Comment:
Did you write a backlog ticket to replace this with the submodelOriginUrl after the fair? Otherwise its inconsistent right?